bundles/binding/org.openhab.binding.rwesmarthome/src/main/java/org/openhab/binding/rwesmarthome/internal/communicator/client/RWEClient.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rwesmarthome.internal.communicator.client;
+
+import java.io.IOException;
+
+import org.openhab.binding.rwesmarthome.internal.communicator.exceptions.RWESmarthomeSessionExpiredException;
+
+/**
+ * @author ollie-dev
+ *
+ */
+public interface RWEClient {
```

Comment:
what is the purpose of this Interface? There doesn't seem to be much infrastructure using this